storage_service/static/js/project.js

```diff
@@ -27,4 +27,34 @@ $(document).ready(function() {
           },
       }
     });
-} );
+
+    $("a.request-delete").click(function() {
+        var self = $(this);
+        var uuid = self.data('package-uuid');
+        var pipeline = self.data('package-pipeline');
+        var $userDataEl = $("#user-data-packages");
+        var userId = $userDataEl.data('user-id');
+        var userEmail = $userDataEl.data('user-email');
+        var userUsername = $userDataEl.data('user-username');
+        var userAPIKey = $userDataEl.data('user-api-key');
+        var formData = {
+            "event_reason": "Storage Service user wants to delete AIP " + uuid + ".",
+            "pipeline": pipeline,
+            "user_id": userId,
+            "user_email": userEmail};
+        $.ajax({
+            type: "POST",
+            url: "/api/v2/file/" + uuid + '/delete_aip/',
+            data: JSON.stringify(formData),
+            dataType: "json",
+            contentType: "application/json; charset=utf-8",
+            headers: {"Authorization": "ApiKey " + userUsername + ":" + userAPIKey},
```

Comment:
Is it the cookie not being sent? AFAIK the SS API supports SessionAuthentication.

Reply:
On my dev machine, the browser is not sending the cookie, presumably because there is no domain being used and the cross-domain no-cookies thing is kicking in. Without adding CORS/OPTIONS logic, I think sending the API key in the headers is the best solution.